docs/how-to/jcloud_deploy.md

```diff
@@ -0,0 +1,72 @@
+# Deploy on JCloud
+
+This tutorial shows how to deploy CLIP Encoder on JCloud. 
+Click [here](https://github.com/jina-ai/jcloud) for more information on JCloud usage.
+A sample YAML file of a [Jina Flow](https://docs.jina.ai/fundamentals/flow/) to run minimum CLIP Encoder is given as
+
+```yaml
+# flow.yml
+jtype: Flow
+executors:
+  - name: CLIPTorchEncoder # The name of the encoder
+    uses: jinahub+docker://CLIPTorchEncoder # You can find more on Jina Hub
```

Comment:
```suggestion
    uses: jinahub+docker://CLIPTorchEncoder # You can find more on Jina Hub???? How? where is it? whats the URL?
```

Reply:
There will be more. Let's remove it for now